docs/configuration.md

```diff
@@ -828,7 +828,13 @@ the `HOST:PORT` on which the debug server should accept connections.
 If the registry is configured as a pull-through cache, the `debug` server can be used
 to access proxy statistics. These statistics are exposed at `/debug/vars` in JSON format.
 
-## `prometheus`
+#### `prometheus`
+
+```none
+    prometheus:
```

Comment:
I think indentation can be removed here, because you're using code-fences now

```suggestion
 prometheus:
```

Reply:
if we are doing this, we should probable move the child elements to left, too 🤔

Reply:
Ah, yes meant all the lines, but was commenting from my phone ☺️